libs/template/application/src/auth/bapi/plugin.ts

```diff
@@ -34,7 +34,7 @@ export class BapiPlugin implements AppPlugin {
             (currentRoute.startsWith(oauthConfig.loginRoute) ||
               currentRoute.startsWith(redirectUrl.pathname))
           ) {
-            resolve(RouterService).navigate('/');
+            setTimeout(() => resolve(RouterService).navigate('/'), 500);
```

Comment:
I don't think it's a good idea to add delay to the app just for E2E tests.
Maybe you can instead intercept and add delay to a network request to achieve the same result without slowing down the app?

Reply:
yep, that's debatable actually... 

from user perspective thinks happen to fast and he might miss that loading screen appearance and be confused

as for me - we should have a delay here (for better UX) but make it like 250-300ms (to just have it as a "blink" but still be able to see loader)

Reply:
We should not introduce artificial delays to the app - if it's fast it should behave fast, the loader screen purpose is to communicate to a user that something is still happening and if there is nothing happening we should not display it just for the sake of it looking "nicer".
However the fact that loader screen pops too fast seems like an implementation bug as it should be visible until after the data has been fully loaded from BAPI into local DB which clearly does not happen, so I would suggest instead to create a bug ticket to fix the root cause rather then adding extra delays to simulate loading.

Reply:
Created a bug ticket for this. Let's remove the delay from this PR and comment out related E2E tests for now until the bug it resolved.